about.html

```diff
@@ -0,0 +1,188 @@
+          ><img src="./images/logo.png" alt=""
+        /></a>
+        <ul class="navbar_container">
+          <li class="navbar_items">
+            <a href="./index.html" class="nav_links">Home</a>
+          </li>
+          <li class="navbar_items">
+            <a href="#" class="nav_links">Progarm</a>
+          </li>
+          <li class="navbar_items">
+            <a href="#" class="nav_links">Join</a>
+          </li>
+          <li class="navbar_items">
+            <a href="#" class="nav_links">Sponsor</a>
+          </li>
+          <li class="navbar_items">
+            <a href="#" class="nav_links">News</a>
+          </li>
+          <li class="navbar_items">
+            <a class="nav-bttn" href="#">UNAMA Campaign</a>
+          </li>
+        </ul>
+      </nav>
+      <div class="hamburger">
+        <span class="lines one"></span>
+        <span class="lines two"></span>
+        <span class="lines three"></span>
+      </div>
+    </header>
+    <!-- Main sectiom  -->
+    <main class="container-fluid main-about">
+      <section class="row justify-content-center">
+        <div class="col-md-11 text-md-center col-11 p-1 m-2 content-wrapper">
+          <p class="short-para text-center">"Afghanistan Development"</p>
+          <h1 class="main-heading-about text-center">
+            Afghanistan <br />
+            Conference Geneva <br />
+            2020
+          </h1>
+          <p class="info-about text-center p-md-5">
+            The 2020 Afghanistan Conference was a pledging conference, co-hosted
+            by the Governments of Afghanistan and Finland with the United
+            Nations, that took place November 2020. The 2020 Afghanistan
+            Conference was a pledging conference, co-hosted by the Governments
+            of Afghanistan and Finland with the United Nations, that took place
+            November 2020.
+          </p>
+          <p class="place-info-about text-center">
+            @ Geneva Finland, Organized by the Government of Afghanistan
+          </p>
+        </div>
+      </section>
+    </main>
+    <section class="row justify-content-center p-md-5 p-5">
+      <div>
+        <h2 class="text-center p-4 p-md-4 pb-md-1">
+          See The Past 2020 Afghanistan Conference
+        </h2>
+        <p class="text-center p-2 p-md-2 past-para">
+          Take a look at the past two 2020 Afghanistan Conference
+        </p>
+      </div>
+      <div class="col-md-4 col-8 text-center p-4">
+        <img class="footer-logo" src="./images/logo.png" alt="" />
+      </div>
+      <div class="col-10 p-3 p-md-3 m-3 m-md-3">
+        <img
+          class="m-3 m-md-3 col-md-5"
+          src="./images/2020-Afghanistan-Conference.webp"
+          alt="Afghanistan 2020 conference photo"
+        />
+        <img
+          class="m-3 m-md-3 col-md-5"
+          src="./images/2020-Afghanistan-Conference2"
+          alt="Afghanistan 2020 conference photo"
+        />
+      </div>
+    </section>
```

Comment:
- [ ] **As suggested by the previous reviewer,** according to the [given design,](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015) there is a missing section on your About page desktop, :joy: 
 The given design has
   - 1) Top section
   - 2) Middle section
   - 3) Bottom section
  
  We are required to replicate the design and only change the contents so it would be better to separate  your middle section with bottom section and try to replicate the given design as match as possible to fulfill the project requirements :pray: 

### Given design :point_down: 
![image](https://user-images.githubusercontent.com/102757126/205455586-2828261d-6590-4955-bd9a-146e25a57e26.png)